Packages/CandidateUI/Sources/CandidateUI/VerticalCandidateController.swift

```diff
@@ -134,6 +134,7 @@ public class VerticalCandidateController: CandidateController {
         scrollView.verticalScrollElasticity = .none
         scrollView.drawsBackground = false
         scrollView.contentView.drawsBackground = false
+        scrollView.contentView.postsBoundsChangedNotifications = true
 
         tableView = NSTableView(frame: contentRect)
```

Comment:
I was wondering if using the `VerticalCandidateTableView` implemented in this file (which has a `adjustScroll` override used in the old, Objective-C version) instead of `NSTableView` would solve the problem, but then I've tested both, and I like the new solution a lot. The legacy solution would effectively "add a lot of resistance" to the scroll (since the user needs to scroll past the row height threshould), whereas this solution allows the user to scroll as few points as they like and then causes the key strip label highlight to catch up. So this LGTM.